README.md

```diff
@@ -1,2 +1,181 @@
 # btp-toolbox
 Tools for building and debugging with the Bilateral Transport Protocol (BTP)
+
+# <img src="./assets/spider.svg" width="25px"> Spider
+
+The BTP Spider can be configured to act as one BTP server and/or one or multiple BTP clients.
```

Comment:
I don't really understand the purpose of this module from this description. Why would I want a BTP server or clients? Isn't a client a plugin? Is the server a connector or a receiver or something else?